test/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	k8s.io/component-base v0.29.2
 	k8s.io/klog/v2 v2.110.1
 	k8s.io/utils v0.0.0-20231127182322-b307cd553661
-	sigs.k8s.io/cluster-api v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api v1.6.3
```

Comment:
let's drop the capi changes here.

They also have no effect because of the replace above and would be a downgrade compared to the above..

I just pushed 

- #2815 

to bump the capi dependency

Reply:
Didn't see that one, I intended to remove replace statement on top, but somehow got added back. 😄  
I will revert CAPI version update.